test/addBuildPhase.js

```diff
@@ -194,4 +194,26 @@ exports.addBuildPhase = {
         test.equal(buildPhase.shellScript, '"echo \\"hello world!\\""');
         test.done();
     },
+    'should add the PBXBuildPhase with alwaysOutOfDate property': function (test) {
+        var options = {
+            shellPath: '/bin/sh',
+            shellScript: 'test',
+            alwaysOutOfDate: true
+        };
+
+        var buildPhase = proj.addBuildPhase([], 'PBXShellScriptBuildPhase', 'Run a script', proj.getFirstTarget().uuid, options).buildPhase;
+        test.equal(buildPhase.shellPath, '/bin/sh');
+        test.equal(buildPhase.shellScript, '"test"');
+        test.equal(buildPhase.alwaysOutOfDate, true);
```

Comment:
The way this test is conducted this will pass but was this tested against a real project?

In my xcode projects, xcode will use `1` or `0` values instead of true and false. I'm not sure if our writer will translate booleans to 0/1 when printed to file, or if it will print as "true", I'm also not sure if xcode will accept a "true" value over `1`.

For the sake of "self-documenting" maybe the test should match the value types that is used by xcode?

Reply:
Also other places in the test will use the integers to represent the boolean as well: https://github.com/apache/cordova-node-xcode/blob/master/test/addBuildPhase.js#L61 so using ints will also make the test more consistent with other parts of the codebase, even if `true`/`false` is accepted.